fisher.fish

```diff
@@ -117,7 +117,7 @@ function $fisher_cmd_name -d "fish plugin manager"
     command mkdir -p {"$fish_path","$fish_config"}/{conf.d,functions,completions} "$fisher_config" "$fisher_cache"
     or return 1
 
-    set -l completions "$fish_config/completions/$fisher_cmd_name.fish"
+    set -l completions "$fish_path/completions/$fisher_cmd_name.fish"
```

Comment:
@jbucaran 

> Fisherman will correctly take care of plugins it installs and make sure functions/completions/snippets are available after they are installed regardless of their location.

Unless I'm mistaken this will only work on initial install if `$fish_path` is set and the user does not manually update `fish_function_path` and `fish_function_path `. The next time they load fish these things will not be available.